Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM funkyfuture/nest-of-serpents
```

Comment:
Without Dockerfile fix, tests ever did not run. WIt this fix tests fail. On any Fedora version. I don't know how to fix